CMakeLists.txt

```diff
@@ -1,7 +1,12 @@
 cmake_minimum_required(VERSION 2.6)
 project(mGBA C)
 set(BINARY_NAME mgba CACHE INTERNAL "Name of output binaries")
-set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall -Wextra -std=c99")
+if(NOT MSVC)
+	set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall -Wextra -std=c99")
+else()
+	set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Dsnprintf=_snprintf -Dinline=__inline")
```

Comment:
Please do this in src/util/common.h, as there are already some defines like this in there.